### DISCUSSION TITLE
Popover create & destroy behavior like v1 in v2

### DISCUSSION THREAD

**Discussion by EverchronGreg** [bootstrap-vue/bootstrap-vue#6549] [Q&A]
It seems that in v1, all components within popovers available on a page would be created and destroyed at the same time as the page. In BV v2 they are now created and destroyed when the popover is shown and hidden, so their contents are re-set each time, and logic within them are not watching the store for changes, or available to receive an $emit, breaking a large amount of functionality in our app.

Would someone be able to advise if there is a way for me to restore the old behavior in v2 or confirm that I simply cannot upgrade? ty

**Comment by Hiws** [comment]
Tooltips/Popovers were completely rewritten in the `2.0.0` release, which might be why you're seeing a difference.
I've never used the v1, so can't say much about the behavior you're used to.

You could try installing `v2.0.0-rc.27`, which is the second to last version before the `2.0.0` release, to see if your tooltip/popovers still behave as they do in v1. If that is the case, then it's most likely the `2.0.0` rewrite which has caused the change.

In any case, you might be out of look as I'm not sure there's a way to achieve what you want.
Your best bet might be to either fork the BootstrapVue repo and modify them to fit your needs, or write your own popovers and use those.
